environment.yml

```
@@ -11,7 +11,7 @@ dependencies:
   - litex-hub::symbiflow-yosys-plugins=1.0.0_7_g59ff1e6_23_g3a95697_17_g00b887b_0194_g40efa51=20201120_145821
   - litex-hub::prjxray-tools=0.1_2697_g0f939808=20201120_145821
   - litex-hub::prjxray-db=0.0_0239_gd87c844=20201120_145821
-  - litex-hub::vtr=v8.0.0_3015_gd912bdb88=20201210_100534
+  - litex-hub::vtr-optimized=v8.0.0_3049_g64d15e2dd=20201216_212525
```

```suggestion
  - litex-hub::vtr-optimized=v8.0.0_3049_g64d15e2dd=20201217_025303
```

AFAIK, only the packages labeled as `main` are available. Using a non-main labeled package results in conda not being able to find it.

---

Fixed. I was still waiting on the `main` package, so I thought I'd try it.